### PR TITLE
Release 4.13.0

### PR DESCRIPTION
**Pull request by jezdez**
### Checklist - did you ...

- [x] Add / update outdated documentation?

### CODE REVIEW

File: .authors.yml [hunk 1]
```diff
@@ -597,17 +618,14 @@
...
-  github: msarahan
 - name: Jacob Hultman
-  email: jhultman@novateurresearch.comgit config --global user.email jhultman@novateurresearch.com
+  email: jhultman@novateurresearch.com
   num_commits: 1
   first_commit: 2019-11-15 18:16:26
   aliases:
   - Jacob
+  alternate_emails:
+  - jhultman@novateurresearch.comgit config --global user.email jhultman@novateurresearch.com
```
Comment (beeankha) [inline .authors.yml:628]: This `alternate_emails` section needs to be removed

File: .authors.yml [hunk 2]
```diff
@@ -424,47 +443,52 @@
...
   first_commit: 2012-10-17 04:24:05
-  alternate_emails:
-  - bryan@ubuntu.(none)
   aliases:
   - bryanv@continuum.io
-- name: srossross
+  - Bryan Van de Ven
+  alternate_emails:
+  - bryanv@continuum.io
+  - bryan@ubuntu.(none)
```
Comment (beeankha) [inline .authors.yml:462]: Should this email address be `bryan@ubuntu.com` (or should we delete this line)?

File: .authors.yml (hunk 1 above)
Comment (kenodegard) [reply .authors.yml:628]: This is needed to correct the bad email used in the git history.

File: .authors.yml (hunk 2 above)
Comment (kenodegard) [reply .authors.yml:462]: Ditto

**Review by beeankha — APPROVED** [review]

**Review by kenodegard — APPROVED** [review]

**Pull request merged**